### PR TITLE
Contribute sidecar image for ruby language server

### PR DESCRIPTION
**Pull request by liningpan**
To be able to install solargraph, which is a ruby language server, I have to use a stretch base image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6.6-stretch
```
Comment (ericwill) [inline Dockerfile:12]: This file needs a copyright header, same as the other files. Other than that LGTM.

**Comment by ericwill** [comment]
@liningpan now you can open a PR against the che-plugin-registry, the image was auto-built here and pushed to `quay.io/eclipse/che-sidecar-ruby:2.6-76f1125`

**Comment by liningpan** [comment]
> 
> 
> @liningpan now you can open a PR against the che-plugin-registry, the image was auto-built here and pushed to `quay.io/eclipse/che-sidecar-ruby:2.6-76f1125`

Are there any plugin naming conventions? Should I put the plugin under something like `ruby/solargraph/`?

**Comment by ericwill** [comment]
Please put the new folder under `/v3/plugins/`. The folder name is the publisher name of the extension you are adding, in this case if it's vscode-solargraph the folder structure would be `/v3/plugins/castwide/<VERSION>`. Please also add an entry into the [vscode-extensions.json file](https://github.com/eclipse/che-plugin-registry/blob/master/vscode-extensions.json). You can read about that file [here](https://github.com/eclipse/che-plugin-registry/blob/master/CONTRIBUTE.md). You can also look at other entries as an example.

**Comment by ericwill** [comment]
If you need any other help just open the PR and I'll be happy to help you with the details.

**Pull request merged**